website/www/site/content/en/contribute/_index.md

```diff
@@ -83,217 +76,136 @@ sudo apt-get install \
    docker-ce
 ```
 
-On some systems (like Ubuntu 20.04) these need to be installed also
-```
-pip3 install grpcio-tools mypy-protobuf
-```
+</br>
 
 You also need to [install Go](https://golang.org/doc/install).
 
+</br>
+
 Once Go is installed, install goavro:
 
+</br>
+
```

Comment:
Can you please avoid adding `</br>` tags to the content?

Reply:
I deleted as much as I could